onboarding-manager/pkg/cloudinit/99_infra.cfg

```diff
@@ -131,6 +131,9 @@ write_files:
       RELEASE_FQDN={{ .RELEASE_FQDN }}
       RS_TYPE={{ .RS_TYPE }}
       RSTYPE={{ .RS_TYPE }}
+      DISABLE_CO_PROFILE={{ .DISABLE_CO_PROFILE }}
+      DISABLE_AO_PROFILE={{ .DISABLE_AO_PROFILE }}
```

Comment:
`DISABLE_AO_PROFILE` is set but not used anywhere, it doesn't make significance in EN side right. Because there is no agent associated with AO. Then should we keep this variable?

Reply:
AO_DISABLE make sense in the orchestrator deployment. But does nothing on the EN provisioning. My point is if it is not used then what is the point to set it